docs/share_usage.md

```diff
@@ -0,0 +1,299 @@
+# File Persona usage guide
+
+The HPE 3PAR File Persona feature allows user to manage file 
+shares on 3PAR arrays through Docker interface.
+
+In order to use HPE 3PAR File Persona feature, user needs to 
+configure a backend one for each target array as below:
+
+## Configuring backend for file share
+
+```sh
+[DEFAULT]
+
+# ssh key file required for driver ssh communication with array
+ssh_hosts_key_file = /root/.ssh/known_hosts
+
+# IP Address and port number of the ETCD instance
+# to be used for storing the share meta data
+host_etcd_ip_address =  10.50.164.1
+host_etcd_port_number = 2379
+
+# Client certificate and key details for secured ETCD cluster
+# host_etcd_client_cert = /root/plugin/certs/client.pem
+# host_etcd_client_key = /root/plugin/certs/client-key.pem
+
+# Logging level for the plugin logs
+logging = DEBUG
+
+# Logging level for 3PAR client logs
+hpe3par_debug = True
+
+# Suppress Requests Library SSL warnings
+suppress_requests_ssl_warnings = True
+
+# Set the driver to be File driver
+hpedockerplugin_driver = hpedockerplugin.hpe.hpe_3par_file.HPE3PARFileDriver
+
+hpe3par_api_url = https://10.50.3.24:8080/api/v1
+hpe3par_username = 3paradm
+hpe3par_password = 3pardata
+hpe3par_san_ip = 10.50.3.24
```

Comment:
I would suggest everywhere we can mask the IP address, but lets have the port number visible.